dessia_common/workflow/core.py

```diff
@@ -1965,6 +1965,20 @@ def to_markdown(self, **kwargs) -> str:
         return template.substitute(name=self.name, workflow_name=self.workflow.name,
                                    output_table=output_table, execution_info=execution_info)
 
+    def zip_settings(self):
+        """ Returns a list of streams that contain different exports of the objects. """
+        streams = []
+        for export_format in self._export_formats():
+            if export_format.extension != "zip":
+                method_name = export_format.method_name
+                stream_class = StringFile if export_format.text else BinaryFile
+                stream = stream_class(filename=f"export.{export_format.extension}")
+                block_index = export_format.args.get('block_index', None)
+                _ = getattr(self, method_name)(stream, block_index) if block_index\
```

Comment:
If only block export formats are in use, the negative part of this test is not needed as every export format should have an index

Reply:
Done !